auth/routes/routes.ts

```diff
@@ -38,7 +55,15 @@ router.get('/organizaciones', Auth.authenticate(), (req, res, next) => {
         if (err) {
             return next(err);
         }
-        const organizaciones = user.organizaciones.map((item) => {
+
+        // user.organizaciones.forEach(element => {
+        //     if (element.permisosPausados === false) {
+        //         organizacionesFiltradas.push(element);
+        //     }
+        // });
+        organizacionesFiltradas = user.organizaciones.filter(x => x.permisosPausados === false);
```

Comment:
```suggestion
        const organizacionesFiltradas = user.organizaciones.filter(x => x.permisosPausados === false);
```
`organizacionesFiltradas` se puede definir directamente acá.